src/EnergyPlus/Plant/Solution/PlantLoopSolver.cc

```diff
@@ -93,18 +93,14 @@ namespace PlantLoopSolver {
 	// Calls various worker routines to model flow rates around a plant half loop
 	// The procedural flow depends on the pump(s), loop side, and operation scheme at the time (and current flow lock?)
 
-	// Using/Aliasing
-
-	// Data
-	// DERIVED TYPE DEFINITIONS
-
 	// MODULE VARIABLE DEFINITIONS
 	Real64 InitialDemandToLoopSetPoint;
 	Real64 CurrentAlterationsToDemand;
 	Real64 UpdatedDemandToLoopSetPoint;
 	Real64 LoadToLoopSetPointThatWasntMet; // Unmet Demand
 	Real64 InitialDemandToLoopSetPointSAVED;
-	int RefrigIndex( 0 ); // Index denoting refrigerant used (possibly steam)
+    bool CurrentLoopSideIsConstantSpeedBranchPumped;
```

Comment:
Indentation went awry all over the place here.  I'll fix that.

This variable, that I will probably place in a better location, is a flag to indicate that the current loopside is constant speed branch pumped.  This flag signals to the solver that the entire load/flow solution may need to be repeated a second time for these loops.